usaspending_api/api_docs/api_documentation/download/list_agencies.md

```diff
@@ -0,0 +1,75 @@
+## [List Agencies](#download-status)
+**Route:** `/api/v2/bulk_download/list_agencies/`
+
+**Method:** `POST`
+
+This route lists all the agencies and the subagencies or federal accounts associated under specific agencies.
+
+### Request example
+
+```
+{
+    "agency": 1,
+}
+```
+
+### Request Parameters Description
+
+* `agency` - agency database id. If provided, populates `sub_agencies` and `federal_accounts`.
+
+### Response (JSON)
+
+When agency is not provided or agency is 0:
+```
+{
+    "agencies": {
+        "cfo_agencies": [
+            {
+                "name": "Department of Agriculture",
+                "toptier_agency_id": 513,
+                "cgac_code": "012"
+            },
+            ...
+        ],
+        "other_agencies": [
+            {
+                "name": "Abraham Lincoln Bicentennial Commission",
+                "toptier_agency_id": 644,
+                "cgac_code": "289"
+            },
+            ...
+        ]
+    },
+    "sub_agencies": [],
+    "federal_accounts": []
+}
+```
+When agencies is provided with an agency id of `2`:
+```
+{
+    "agencies": [],
+    "sub_agencies": [
+        {
+            "subtier_agency_name": "The Legislative Branch"
+        }
+    ],
+    "federal_accounts": [
+        {
+            "federal_account_name": "United States Capitol Police Memorial Fund, Capitol Police",
+            "federal_account_id": 90
+        },
+        ...
+    ]
+}
+```
+
+The response includes 3 possible objects
+* `agencies` - list of all agencies, provided if no `agency` is provided in the request
+    * `name`
+    * `toptier_agency_id` - database id
+    * `cgac_code` - cgac_code (more for the backend usage)
```

Comment:
Needs a more detailed explanation in case the user isn't familiar. 
`a unique code used by the government to identify agencies (Common Government-wide Accounting Classification)`

Reply:
Done